pkg/cmd/cli/cmd/importimage.go

```diff
@@ -75,11 +75,15 @@ type ImportImageOptions struct {
 
 // Complete turns a partially defined ImportImageOptions into a solvent structure
 // which can be validated and used for aa import.
-func (o *ImportImageOptions) Complete(f *clientcmd.Factory, args []string, out io.Writer) error {
+func (o *ImportImageOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, args []string, out io.Writer) error {
 	if len(args) > 0 {
 		o.Target = args[0]
 	}
 
+	if !cmd.Flags().Lookup("insecure").Changed {
```

Comment:
this probably deserves a comment about the image stream annotation and precedence